lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2565,10 +2565,29 @@ class ConvertAtenGridSamplerOp : public OpConversionPattern<AtenGridSamplerOp> {
       return res;
     };
 
+    auto lambdaBorder = [&](OpBuilder &b, Location loc, Value x,
+                            Value SizeSubOne) -> Value {
+      Value xMaxZero = b.create<arith::MaximumFOp>(loc, x, zeroFloat);
+      return b.create<arith::MinimumFOp>(loc, xMaxZero, SizeSubOne);
+    };
+
+    auto lambdaPadding = [&](OpBuilder &b, Location loc, Value paddingMode,
+                             Value x, Value SizeSubOne) -> Value {
```

Comment:
`x` and `sizeSubOne` should be the only inputs for both of these lambda functions. The others can be gotten through the default capture. Just be sure to define these functions after you introduce the other arguments.

Reply:
If we want to capture the other arguments, the lambdas will have to be defined inside the GenericOp lambda later on the code, but I would want to avoid that as it would clutter the implementation there. There are a few other helper lambdas such as `lambdaInterpolate` and `lambdaExtract` which take arguments like this, so I decided to make it similar to those.